src/auth/ui/vue/backend/types.ts

```diff
@@ -0,0 +1,6 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export type ServiceItemId = 'DOCUMENT_TYPE_SUPPORT' | 'RESOURCE_EXPLORER' | 'CODE_WHISPERER' | 'CODE_CATALYST'
```

Comment:
You should be able to combine this w/ the frontend types. It's ok to use the same file as long as the code is runtime agnostic (node vs. browser). Types are always safe to share.

Reply:
Thanks, moved out of the "backend" folder in to the root